extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -118,13 +121,81 @@
   transform-origin: right !important;
 }
 
+[info] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: 48px !important; 
+}
+
+[info] [dir=rtl] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: auto !important; 
+  padding-left: 48px !important; 
+}
+
+.i-amphtml-message-container {
+  width: max-content !important;
+  transition-property: opacity, transform !important;
+  float: right !important;
```

Comment:
Do we need this float right? Because the element is `position: absolute;`, this float statement should have no effect. Could you try removing it?
Also, because we now support RTL languages, we should try to avoid using floats as they require a CSS override, when flexbox handles it automatically.

Reply:
Removing it works, and yeah that makes sense!